tests/test_optimizely.py

```diff
@@ -2046,6 +2046,134 @@ def test_get_feature_variable__returns_none_if_unable_to_cast(self):
 
     mock_client_logger.error.assert_called_with('Unable to cast value. Returning None.')
 
```

Comment:
It looks like our test cases are (mostly) grouped by API. Can we collate the new tests appropriately?